spec/services/amr/n3rgy_tariffs_download_and_upsert_spec.rb

```diff
@@ -45,6 +50,17 @@ module Amr
          .and change { TariffStandingCharge.count }.by(0)
         expect(TariffImportLog.last.error_messages).to be_blank
       end
+
+      it "should request 24 hours of data and upsert a new tariff price and tariff standing charge if there are no existing records of the same date" do
```

Comment:
think this needs renaming as its testing handling of zero tariffs?

Reply:
Although, as the changes are part of the n3rgy tariff upserter, maybe you should be adding specs to n3rgy_tariffs_upserter_spec.rb instead of, or in additon to the download and upserter?